scripts/utils/create_package_template.py

```diff
@@ -667,6 +675,9 @@ def main(argv=None):
     # remove type before passing to template create function
     del args.type
 
+    # fixup the files argument
+    args.files = ", ".join(repr(name.strip()) for name in args.files.split(","))
```

Comment:
This fails if `files` is not provided (it is not required). I purpose to move this code inside `create_ida_plugin_template` (after checking if `files` is empty) as it is only used by that function.